templates/web/common/dialog_export.html.twig

```diff
@@ -178,9 +178,10 @@
                         <div>
                             <span style="min-width: 21px;display: inline-block; ">
                                 {{ 'export::mail: destinataire' | trans }}</span>
-                            <input type="text" value="" name="destmail" class="required span4">
+                            <input type="text" value="" class="span4 tagsinput" data-role="tagsinput" >
+                            <input type="hidden" value="" name="destmail" class="required span4 destmail" >
```

Comment:
could we use the taginput directly without converting to destmail ? We can change the back to accept ',' or ';', ' ', ... as separators.

Reply:
Ok noted, in that case we need to use `,`  as separators on back